cves/kernel/CVE-2020-11725.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  NOT A SERIOUS VULNERABILITY... The report states that a variable stored as 
+  info->owner (C code) can be misused to cause "interesting side effects" ie.
+  possible elevation of privilege.  However, every usage of the 
+  variable is wrapped in functions that are designed to keep the misuse from
+  breaking existing user privileges.
```

Comment:
A very interesting take, but after looking through some of the sources I agree with the description.